src/components/CreatePost.tsx

```diff
@@ -0,0 +1,21 @@
+import { useUser } from "@clerk/nextjs";
+
+const CreatePost = () => {
+  const { user } = useUser(); 
+  if (!user) return null;
+
+  return (
+    <div className="flex gap-4 w-100">
```

Comment:
```suggestion
    <div className="flex gap-4 w-full">
```
もしこちらがElementのWidthを100%にしたい場合はこちらの書き方が正しいかもしれません！

よければこちらでご確認ください！

[Taikwind Cheat sheet](https://nerdcave.com/tailwind-cheat-sheet)